modular_skyrat/modules/modular_vending/code/vending.dm

```diff
@@ -14,8 +14,38 @@
 	if(skyrat_contraband)
 		contraband += skyrat_contraband
 
+	/// Time to make clothes amounts consistent!
+	for (var/item in products)
```

Comment:
If you're going to do an `ispath(item, /obj/item/clothing)` anyway, then you should do your for loop like this instead, and remove the `ispath` check from that if() statement, as it's going to do that type check for you and it's only going to take out the clothing items from `products`:
```suggestion
	for (var/obj/item/clothing/item in products)
```

Reply:
Ah, nice, I'm used to java where it throws a fit if you don't perfectly sanitize every little thing- force of habit!